clusterloader/docs/design.md

```diff
@@ -278,70 +276,78 @@ be modifying that part of Cluster Loader codebase. Within the codebase, we will
 to provide a relatively easy framework to achieve it though.
 
 At the high level, to implement gathering a given portion of data or measure a new
-SLO, you will need to implement a very simple Go interface, similar to:
+SLO, you will need to implement a very simple Go interface:
 
 ```
 type Measurement interface {
-	StartMeasurement(config MeasurementConfig)
-	StopMeasurement()
+	Execute(config MeasurementConfig) error
```

Comment:
`Execute(config *MeasurementConfig) error`?
I know it's nitpicking, but i want to use this interface (as it is) in code.